lib/make_render.js

```diff
@@ -44,11 +44,12 @@ function makeCreateState(main, can){
 
 		if(!attr.get(state, "statusCode") &&
 				!isEmptyObject(can.route.routes)) {
-			if(!params.route) {
+			// fix: support root-url (i.e '/') in production-mode
+			if(params.route || (params.route === "")) {
```

Comment:
What is the `route` when there isn't a match? I can check this, guessing it is undefined or null or something.